source/javascripts/na/create_test_merchant_account.js

```diff
@@ -22,6 +22,17 @@ $(function() {
         }
     };
 
+    // https://stackoverflow.com/questions/901115/how-can-i-get-query-string-values-in-javascript/901144#901144
+    // get query variable value by name
+    var getParameterByName = function getParameterByName(name, url = window.location.href) {
```

Comment:
Unless there's a specific reason to define this function only once the variable getParameterByName is accessed, the function should simply be defined without the variable.
https://sentry.io/answers/difference-between-var-functionname-function-and-functionname-in-javascript/

Reply:
Change done